app/src/main/java/org/schabi/newpipe/player/helper/PlayerHelper.java

```diff
@@ -382,8 +383,11 @@ public static void setScreenBrightness(@NonNull final Context context,
     public static boolean globalScreenOrientationLocked(final Context context) {
         // 1: Screen orientation changes using accelerometer
         // 0: Screen orientation is locked
+        // if the accelerometer sensor is missing completely, assume locked orientation
         return android.provider.Settings.System.getInt(
-                context.getContentResolver(), Settings.System.ACCELEROMETER_ROTATION, 0) == 0;
+                context.getContentResolver(), Settings.System.ACCELEROMETER_ROTATION, 0) == 0
```

Comment:
I wonder what the `getInt` returns when there is no accelerometer for rotation... Shouldn't it already return 0, as it's the default value passed for when the stored value is not valid?

Reply:
the setting exists in aosp. The manufacturer would have to deliberately remove the setting. So it most likely is there. And if not: all the guys with their realy odd hardware are happy, that the button is there for them